loader/normalize.go

```diff
@@ -74,7 +78,6 @@ func normalize(project *types.Project, resolvePaths bool) error {
 			if _, err := os.Stat(localContext); err == nil {
 				if resolvePaths {
 					s.Build.Context = localContext
-					s.Build.Dockerfile = absPath(localContext, s.Build.Dockerfile)
```

Comment:
This is related to https://github.com/docker/compose/issues/8760

Actually the `Dockerfile` part is just the name of the file and is always relative to `Context`

Reply:
right, but `docker compose config` uses `WithResolvedPaths(false)`
https://github.com/docker/compose/blob/v2/cmd/compose/convert.go#L106, so you should better double check this flag is well passed to `false`, otherwise this won't help

Reply:
Yep. The idea here is to alsopass that to true.